gcloud/storage/acl.py

```diff
@@ -409,7 +419,7 @@ def _save(self, acl, predefined, client):
 
         :type predefined: string or None
         :param predefined: An identifier for a predefined ACL.  Must be one
-                           of the keys in :attr:`_PREDEFINED_ACLS`
+                           of the keys in :attr:`_PREDEFINED_JSON_ACLS`
```

Comment:
Worth making this public if we refer to it? Also shouldn't it be `:data:` instead of `:attr:`?